XenAdmin/Dialogs/DuplicateTemplateNameDialog.ja.resx

```diff
@@ -253,6 +253,6 @@
     <value>DuplicateTemplateNameDialog</value>
   </data>
   <data name="&gt;&gt;$this.Type" xml:space="preserve">
-    <value>XenAdmin.Dialogs.XenDialogBase, XenCenterMain, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null</value>
+    <value>XenAdmin.Dialogs.XenDialogBase, XenCenter, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null</value>

```

Comment:
In PR #1803 we changed "XenCenter" to "XenCenterMain" in some of the resource files. Please make sure that we are not regressing that commit.